app/helpers/application_helper.rb

```diff
@@ -9,4 +11,13 @@ def alert_class_for(flash_type)
     }
     types[flash_type.to_sym]
   end
+
+  def all_months_select_options(first_month, last_month, end_of_month_flg = false)
+    all_months(first_month, last_month, end_of_month_flg).map { |month| [month.strftime('%Y年%m月'), month] }
+  end
+
+  def all_months(first_month, last_month, end_of_month_flg = false)
```

Comment:
__[reek]__
* [BooleanParameter] has boolean parameter 'end_of_month_flg' [:ref](https://github.com/troessner/reek/blob/master/docs/Boolean-Parameter.md)
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)

<!-- Comment by [SideCI](https://sideci.com) -->